src/components/button/button.stories.mdx

```diff
@@ -214,6 +214,68 @@ Dark Background buttons can be set to `noWrap` mode to prevent text wrapping.
   <Story name="darkBackground/noWrap" story={stories.DarkBackgroundButtonNoWrap} />
 </Canvas>
 
+### Gradient Buttons
```

Comment:
**comment** (non-blocking): I don't think there was a need to add so many examples of both the new "gradient" types. Personally I think a single example of each, or perhaps 2 (one with and one without an icon) was sufficient - the rest (`disabled` etc) are just combinations of props that consumers would expect to work anyway, we don't need to have a story for every possible combination of props imo!

If this was done for Chromatic then we could move some of these to test stories?

But not a blocker, I don't object to leaving all these in if you tihnk it's best.

Reply:
yeah I just matched the other variants tbh, my thinking is that once we decide on how we want to structure stories going forward they would all match. I also wanted to make it clear that it supports these but not `destructive` but I think I can achieve that by adding some more text to the main stories. Happy to move these to the test file if the other reviewer agrees